### PR TITLE
Arcadia 8

### PR DESCRIPTION
**Pull request by Billiam9420**
MCDM's Arcadia Issue 8 (BREW-429)

### CODE REVIEW

File: book/MCDM Productions; Arcadia Issue 8.json [hunk 1]
```diff
@@ -0,0 +1,3593 @@
...
+            "className": "Warlock",
+            "classSource": "PHB",
+            "subclassSpells": [
+                "vicious mockery"
+            ],
+            "additionalSpells": [
+                {
+                    "known": {
+                        "0": [
+                            "vicious mockery"
```
Comment (Git-GoR) [inline book/MCDM Productions; Arcadia Issue 8.json:49]: Tag cantrips in `additionalSpells` like that
```suggestion
                            "vicious mockery#c"
```

File: book/MCDM Productions; Arcadia Issue 8.json [hunk 2]
```diff
@@ -0,0 +1,3593 @@
+{
+    "$schema": "https://raw.githubusercontent.com/TheGiddyLimit/homebrew/master/_schema-fast/homebrew.json",
```
Comment (Git-GoR) [inline book/MCDM Productions; Arcadia Issue 8.json:2]: The `$schema` key is VSCode-exclusive, and while it's very useful during testing it's better to remove them before uploading to the repo
```suggestion
```

File: book/MCDM Productions; Arcadia Issue 8.json [hunk 3]
```diff
@@ -0,0 +1,3593 @@
...
+                        "col-2 text-center",
+                        "col-2 text-center"
+                    ],
+                    "rows": [
+                        [
+                            "1st",
+                            "{@spell detect magic}, {@spell silent image}",
+                            "{@spell magic missile}, {@spell sleep}",
+                            "{@spell create or destroy water}, {@spell fog cloud}",
+                            "{@spell animal friendship}, {@spell tasha's hideous laughter|hideous laughter}"
```
Comment (Git-GoR) [inline book/MCDM Productions; Arcadia Issue 8.json:178]: With `@spell` tags, the pipe order is `name|source|optional text`. So as you had it, you were linking to the `tasha's...` spell with the `hideous laughter` source.

(Be aware that spell tags default to the `PHB` source by default, so doing `{@spell tasha's hideous laughter||hideous laughter` is also possible. I just wrote it to make it more clear.
```suggestion
                            "{@spell animal friendship}, {@spell tasha's hideous laughter|PHB|hideous laughter}"
```

File: book/MCDM Productions; Arcadia Issue 8.json [hunk 4]
```diff
@@ -0,0 +1,3593 @@
...
+                            "{@spell  bestow curse}, {@spell conjure animals}",
+                            "{@spell blink}, {@spell slow}",
+                            "{@spell water walk}, {@spell wind wall}",
+                            "{@spell lightning bolt}, {@spell leomund's tiny hut}"
+                        ],
+                        [
+                            "4th",
+                            "{@spell dominate beast}, {@spell polymorph}",
+                            "{@spell faithful hound}, {@spell phantasmal killer}",
+                            "{@spell evard's black tentacles|black tentacles}, {@spell control water}",
```
Comment (Git-GoR) [inline book/MCDM Productions; Arcadia Issue 8.json:198]: Same thing as before.
```suggestion
                            "{@spell evard's black tentacles||black tentacles}, {@spell control water}",
```

File: book/MCDM Productions; Arcadia Issue 8.json [hunk 5]
```diff
@@ -0,0 +1,3593 @@
...
+                                    "href": {
+                                        "type": "external",
+                                        "url": "https://i.imgur.com/HaIaJM7.jpg"
+                                    },
+                                    "title": "Optional Title"
+                                },
+                                {
+                                    "type": "list",
+                                    "items": [
+                                        "{@class Warlock|Warlock Hag Patron|Hag|Ar8}"
```
Comment (Git-GoR) [inline book/MCDM Productions; Arcadia Issue 8.json:1232]: Class tags: `{@class className|classSource|text|subclassShortName|subclassSource}`
```suggestion
                                        "{@class Warlock|PHB|Hag|Hag|Ar8}"
```

File: book/MCDM Productions; Arcadia Issue 8.json [hunk 6]
```diff
@@ -0,0 +1,3593 @@
...
+                                    "type": "image",
+                                    "href": {
+                                        "type": "external",
+                                        "url": "https://i.imgur.com/YWMiWvC.jpg"
+                                    }
+                                },
+                                {
+                                    "type": "list",
+                                    "items": [
+                                        "{@class Sorcerer|Wicked Witch Sorcerous Origin|Wicked Witch|Ar8}"
```
Comment (Git-GoR) [inline book/MCDM Productions; Arcadia Issue 8.json:1252]: Same as before
```suggestion
                                        "{@class Sorcerer|PHB|Wicked Witch|Wicked Witch|Ar8}"
```

File: book/MCDM Productions; Arcadia Issue 8.json [hunk 7]
```diff
@@ -0,0 +1,3593 @@
...
+                                    "type": "image",
+                                    "href": {
+                                        "type": "external",
+                                        "url": "https://i.imgur.com/tAEPuAS.jpg"
+                                    }
+                                },
+                                {
+                                    "type": "list",
+                                    "items": [
+                                        "{@class Bard|Bardic College of Grandmother's Tales|Grandmother's Tales|Ar8}"
```
Comment (Git-GoR) [inline book/MCDM Productions; Arcadia Issue 8.json:1272]: Same as before
```suggestion
                                        "{@class Bard|PHB|Grandmother's Tales|Grandmother's Tales|Ar8}"
```

File: book/MCDM Productions; Arcadia Issue 8.json [hunk 8]
```diff
@@ -0,0 +1,3593 @@
...
+                            "name": "Warlock Hag Patron",
+                            "page": 3,
+                            "entries": [
+                                {
+                                    "type": "image",
+                                    "href": {
+                                        "type": "external",
+                                        "url": "https://i.imgur.com/HaIaJM7.jpg"
+                                    },
+                                    "title": "Optional Title"
```
Comment (Git-GoR) [inline book/MCDM Productions; Arcadia Issue 8.json:1227]: I doubt the title of that image is `Optional Title` 😛
```suggestion
                                    }
```

File: book/MCDM Productions; Arcadia Issue 8.json [hunk 9]
```diff
@@ -0,0 +1,3593 @@
...
+                                },
+                                {
+                                    "type": "entries",
+                                    "name": "{@creature Aboleth}",
+                                    "page": 15,
+                                    "entries": [
+                                        "These aboleth components provide the following benefits.",
+                                        "{@b {@i Aboleth Eye.}} When you cast {@spell legend lore} using this component, you can learn about an additional person, place, or object of legendary importance.",
+                                        "{@b {@i Piece of Aboleth Brain.}} When you cast a spell that deals psychic damage using this component, the spell’s damage die size increases by one: a d4 becomes a d6, a d6 becomes a d8, and so on, to a maximum of a d12. You also deal three extra dice of damage whenever the spell deals damage.",
+                                        "{@b {@i Piece of Aboleth Tentacle.}} When you cast {@spell evard's black tentacles|black tentacles} using this component, a creature that takes damage from the tentacles takes an additional 1d10 acid damage."
```
Comment (Git-GoR) [inline book/MCDM Productions; Arcadia Issue 8.json:1488]: Same as before
```suggestion
                                        "{@b {@i Piece of Aboleth Tentacle.}} When you cast {@spell evard's black tentacles||black tentacles} using this component, a creature that takes damage from the tentacles takes an additional 1d10 acid damage."
```

File: book/MCDM Productions; Arcadia Issue 8.json [hunk 10]
```diff
@@ -0,0 +1,3593 @@
...
+                                "{@b {@i Green with Envy.}} The characters learn the truth about the Magia Hand and Emerald, and she needs them to disappear. Quickly. Alternatively, their success retrieving items for Emerald raises her suspicions—she no longer trusts that her secret will remain hers in the face of close, skilled adventurers, and she’s unwilling to risk them discovering her past or the existence of the Magia Hand.",
+                                "Emerald makes a final request of the characters: she tells them that she has arranged to pick up a {@item figurine of wondrous power|PHB} from another adventuring group, but she needs the characters to scout the handoff location first. This is a believable lie—the other group does have the figurine, but they are {@creature assassin|MM|assassins} from the Magia Hand who ambush the characters. If the characters survive, the Emerald Exchange remains closed and Emerald disappears...for now."
```
Comment (Git-GoR) [inline book/MCDM Productions; Arcadia Issue 8.json:3531]: Wrong source
```suggestion
                                "Emerald makes a final request of the characters: she tells them that she has arranged to pick up a {@item figurine of wondrous power|DMG} from another adventuring group, but she needs the characters to scout the handoff location first. This is a believable lie—the other group does have the figurine, but they are {@creature assassin|MM|assassins} from the Magia Hand who ambush the characters. If the characters survive, the Emerald Exchange remains closed and Emerald disappears...for now."
```

**Review by TheGiddyLimit — APPROVED** [review]

**Pull request merged**